exporter/awsemfexporter/metric_translator_test.go

```diff
@@ -256,6 +256,18 @@ func assertCWMeasurementEqual(t *testing.T, expected, actual CWMeasurement) {
 	assertDimsEqual(t, expected.Dimensions, actual.Dimensions)
 }
 
+func TestGetNamespace(t *testing.T) {
```

Comment:
Can you add the following test cases:
1. `config.Namespace` is non-empty
2. `config.Namespace` and service namespace is also empty
3. Each of the 3 if clauses (i.e. service name and service namespace are strings, only service name is string, only service namespace is string)